README.md

```diff
@@ -18,6 +18,8 @@ To get started with RIBs, please refer to the [RIBs documentation](https://githu
 
 To get more hands on with RIBs, we have written a [series of tutorials](https://github.com/uber/RIBs/wiki) that run you through the main aspects of the architecture with hands-on examples.
 
+To read about the backstory on why we created RIBs, see [this blog post](https://eng.uber.com/new-rider-app/) we wrote when releasing RIBs in production the first time.
```

Comment:
We also have an Uber mobility video on RIBs, not sure if the blog post references the video so might be nice to include a link also.

https://www.youtube.com/watch?v=Q5cTT0M0YXg

Reply:
Router, Interactor **AND** Builder?

Reply:
Thanks, both! Addressed both of them.